lib/ansible/plugins/filter/split.yml

```diff
@@ -23,7 +23,7 @@ EXAMPLES: |
     listjojo: "{{ 'jojo is a' | split }}"
 
     # listjojocomma => [ "jojo is", "a" ]
-    listjojocomma: "{{ 'jojo is, a' | split(',') }}"
+    listjojocomma: "{{ 'jojo is, a' | split(',' ) }}"
```

Comment:
As you can see in the diff that was created, this was already fixed in 0f01c69f1e2528b935359cfe578530722bca2c59.